apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeCard.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Box,
+  Button,
+  Card,
+  FormControl,
+  Select,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { css } from 'emotion';
+import { AllContentTypes, ContentTypeEntries } from 'types';
+
+interface Props {
+  allContentTypes: AllContentTypes;
+  contentTypeEntries: ContentTypeEntries;
+  onContentTypeChange: (prevKey: string, newKey: string) => void;
+  onContentTypeFieldChange: (key: string, field: string, value: string) => void;
+  onAddContentType: () => void;
+  onRemoveContentType: (key: string) => void;
+}
+
+const styles = {
+  contentTypeItem: css({
+    flex: 4,
+  }),
+  removeItem: css({
+    flex: 1,
+  }),
+};
+
+const AssignContentTypeCard = (props: Props) => {
+  const {
+    allContentTypes,
+    contentTypeEntries,
+    onContentTypeChange,
+    onContentTypeFieldChange,
+    onAddContentType,
+    onRemoveContentType,
+  } = props;
+
+  return (
+    <Card>
+      {Object.keys(contentTypeEntries).length ? (
```

Comment:
The only thing I would consider here is two things (made in two separate comments). 
You don't have to go crazy on refactoring the repeatability here but I think there's some low hanging fruit like
```
          <Box className={styles.contentTypeItem}>
            <FormControl marginBottom="none">
              <FormControl.Label>URL prefix</FormControl.Label>
            </FormControl>
          </Box>
```

You can either make this a util component or just have it in this file that takes in some label params and whatever else it needs. 

I have a comment below that shows what I think isn't low hanging fruit and probably isn't worth refactoring although you're totally free to if you think it's useful

Reply:
yup, this is a good callout, thank you! Updating this now